src/testers/pin/check_semantics.py

```diff
@@ -119,6 +120,18 @@ def cafter(instruction):
                 print("     Native address   : %016x" %(nativeAddress))
                 print("     Symbolic address : %016x" %(astAddress))
 
+    # Check destination memory accesses
+    for mem, sym_mem in instruction.getStoreAccess():
```

Comment:
Add check for destination memory access - to be able to verify that instruction writes to the memory correct values.